regression/cbmc/address_space_size_limit1/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE thorough-paths broken-smt-backend
+KNOWNBUG thorough-paths broken-smt-backend
```

Comment:
I believe the "address_space_size_limit*" tests should simply be removed as part this PR.

Reply:
Of course.